domain/schema/model/sql/0016-status.sql

```diff
@@ -0,0 +1,14 @@
+-- Status values for unit and application workloads.
+CREATE TABLE workload_status_value (
+    id INT PRIMARY KEY,
+    status TEXT NOT NULL
+);
+
+INSERT INTO workload_status_value VALUES
+(0, 'unset'),
+(1, 'unknown'),
+(2, 'maintenance'),
+(3, 'waiting'),
+(4, 'blocked'),
+(5, 'active'),
+(6, 'terminated');
```

Comment:
Does agent status or cloud container status need an equivalent?

Reply:
No, this is because we're going to reuse this for application status, they have the same values.

Reply:
It's workload status for both unit and application.